### ISSUE TITLE
 Example_dendrite.tif corrupted // Environment not solvable on MacOS

### ISSUE THREAD

**Issue by lukasjarzembowski** [spbstu-applied-math/SpineTool.Software#1]
Hello! Thank your for creating and sharing this interesting package! I would really love to give this package a try one some of my dendrite data, unfortunately I've had trouble installing and running the example notebook.

The example_dendrite.tif file seems to be corrupted. It's basically 0kb and Fiji cannot open it  "[ERROR] File has length 0 and may be corrupt", which gives rise to an errorwhen running the code `TiffFileError: not a TIFF file`. 

I know it was probably not written on a MacOS system and says so that the requirements are a Windows system but many people use MacOS for their work computers so it would be great if I could install the environment there. Unfortunately, I cannot solve the conda environment on base and conda-forge channels for the packages using the requirements.txt. I've changed the platform parameter to macos-arm, created an environment manually with Python 3.8 and tried to install the required packages using pip instead (didn't work), using a different solver (mamba) etc. - maybe providing exporting the conda environment as .yml would help? (I could install it in the meantime on a windows laptop).

Edit: I exported the environment.yml from my windows machine and tried to install the environment on my mac from that but that environment also could not be solved.

Thank you and best wishes

**Comment by lukasjarzembowski** [comment]
Found the uncorrupted exmaple_dendrite.tif file in this fork https://github.com/DariaWelt/dendritic-spine-shape-analysis/tree/main/example_dendrite!

**Comment by DariaWelt** [comment]
Hi, @lukasjarzembowski 

First of all, sorry for the very late reply — and thank you again for your detailed feedback and for digging into the setup back then.

We’re glad you managed to find a working TIFF file. Regarding macOS: the main difficulty is CGAL, which we use as precompiled binaries. It has to be built specifically for each platform and Python version.

You might find this thread helpful:
https://github.com/spbstu-applied-math/SpineTool.Software/issues/2

There we shared a CGAL build compiled for macOS (Python 3.8) for internal research use. It’s not officially supported, but it may work if your macOS setup is similar (especially Python version and architecture). We also included updated requirements there.

We definitely see the demand for macOS support and will share updates if we’re able to release an officially supported cross-platform version in the future.

Thanks again for your interest in SpineTool!
